repo/blob/s3/s3_options.go

```diff
@@ -19,9 +19,13 @@ type Options struct {
 	DoNotVerifyTLS bool   `json:"doNotVerifyTLS,omitempty"`
 	RootCA         []byte `json:"rootCA,omitempty"`
 
-	AccessKeyID     string `json:"accessKeyID"`
-	SecretAccessKey string `json:"secretAccessKey" kopia:"sensitive"`
-	SessionToken    string `json:"sessionToken"    kopia:"sensitive"`
+	AccessKeyID     string            `json:"accessKeyID"`
+	SecretAccessKey string            `json:"secretAccessKey" kopia:"sensitive"`
+	SessionToken    string            `json:"sessionToken" kopia:"sensitive"`
+	Tags            map[string]string `json:"tags" kopia:"sensitive"`
+	RoleARN         string            `json:"roleARN"`
+	SessionName     string            `json:"sessionName"`
+	RoleDuration    string            `json:"duration"`
```

Comment:
Currently trying out an alternative to make this a `time.Duration` here.
If it works, I'll push it in a separate PR.

Reply:
yea, I was just looking at that. I sort of just pulled this off the branch we had and pushed it. A `time.Duration` definitely makes more sense

Reply:
The issue is that JSON marshaling for `time.Duration` is not that usable.

For example:

```
d := 2*time.Hour + 3 + time.Minute

b, err := json.Marshal(d)

fmt.Printf("%s", b)
// output: 7260000000003
```


PTAL at #4194